### PR TITLE
social nav readme

### PR DESCRIPTION
**Pull request by jimmytyyang**
## Motivation and Context

 This is needed to reproduce what we did for social nav in habitat3 paper.

## How Has This Been Tested

 The social nav training has been verified in habitat `main`

## Types of changes

- **\[Docs change\]** Addition or changes to the documentation

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-baselines/README.md [hunk 1]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+    habitat.task.slack_reward=-0.1 \
+    habitat.environment.max_episode_steps=1500 \
+    habitat.simulator.kinematic_mode=True \
+    habitat.simulator.ac_freq_ratio=4 \
+    habitat.simulator.ctrl_freq=120 \
+    habitat.simulator.agents.agent_0.joint_start_noise=0.0
+```
+
+It is expected to observe the following reward training (learning) curve:
+![Social Nav Reward Trianing Curve](/res/img/habitat3_social_nav_training_reward.png) In addition, under the following slurm job batch script setting:
```
Comment (0mdc) [inline habitat-baselines/README.md:142]: Typo: `Trianing`.

File: habitat-baselines/README.md [hunk 2]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+```bash
+#SBATCH --gres gpu:4
+#SBATCH --cpus-per-task 10
+#SBATCH --nodes 1
+#SBATCH --ntasks-per-node 4
+#SBATCH --mem-per-cpu=6GB
+```
+we have the following training wall clock time versus reward:
+![Social Nav Reward Trianing Curve versus Time](/res/img/habitat3_social_nav_training_reward_time.png)
```
Comment (0mdc) [inline habitat-baselines/README.md:151]: Same as above.

File: habitat-baselines/README.md [hunk 3]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+
+We have the following training FPS:
+![Social Nav Trianing FPS](/res/img/habitat3_social_nav_training_fps.png)
```
Comment (0mdc) [inline habitat-baselines/README.md:154]: Same as above.

File: habitat-baselines/README.md [hunk 4]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+
+It is expected to get the following numbers or something similar after running the evaluation:
+
+```bash
+Average episode social_nav_reward: 1.8821
+Average episode social_nav_stats.has_found_human: 0.9020
+Average episode social_nav_stats.found_human_rate_after_encounter_over_epi: 0.6423
+Average episode social_nav_stats.found_human_rate_over_epi: 0.4275
+Average episode social_nav_stats.frist_ecnounter_steps: 376.0420
+Average episode social_nav_stats.follow_human_steps_after_frist_encounter: 398.6340
```
Comment (0mdc) [inline habitat-baselines/README.md:212]: Typo: `frist`.

File: habitat-baselines/README.md [hunk 5]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+```
+
+It is expected to get the following numbers or something similar after running the evaluation:
+
+```bash
+Average episode social_nav_reward: 1.8821
+Average episode social_nav_stats.has_found_human: 0.9020
+Average episode social_nav_stats.found_human_rate_after_encounter_over_epi: 0.6423
+Average episode social_nav_stats.found_human_rate_over_epi: 0.4275
+Average episode social_nav_stats.frist_ecnounter_steps: 376.0420
```
Comment (0mdc) [inline habitat-baselines/README.md:211]: Typo: `frist` and `ecnounter`

File: habitat-baselines/README.md [hunk 6]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+Average episode social_nav_stats.avg_robot_to_human_after_encounter_dis_over_epi: 1.4969
+Average episode social_nav_stats.avg_robot_to_human_dis_over_epi: 3.6885
+Average episode social_nav_stats.backup_ratio: 0.1889
+Average episode social_nav_stats.yield_ratio: 0.0192
+Average episode num_agents_collide: 0.7020
+```
+
+### Checkpoint
+
+We release a [checkpoint](https://arxiv.org/abs/2106.14405) based on the above command.
```
Comment (0mdc) [inline habitat-baselines/README.md:222]: I would rephrase this as: `We have released a checkpoint...` or `A checkpoint is available here...`.

File: habitat-baselines/README.md [hunk 7]
```diff
@@ -80,11 +80,147 @@ First download the necessary data with `python -m habitat_sim.utils.datasets_dow
...
+    habitat.simulator.ac_freq_ratio=4 \
+    habitat.simulator.ctrl_freq=120 \
+    habitat.simulator.agents.agent_0.joint_start_noise=0.0 \
+    habitat_baselines.load_resume_state_config=False \
+    habitat_baselines.test_episode_count=500 \
+    habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor.height=1080 \
+    habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor.width=1920
+```
+
+It is expected to get the following numbers or something similar after running the evaluation:
```
Comment (0mdc) [inline habitat-baselines/README.md:204]: I would rephrase `It is expected to get the following numbers or something similar...` as `The evaluation is expected to produce values similar to those below:`

**Review by 0mdc — APPROVED** [review]
LGTM! Left some minor comments.

File: habitat-baselines/README.md (hunk 1 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:142]: Thank you!

File: habitat-baselines/README.md (hunk 3 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:154]: Thank you!

File: habitat-baselines/README.md (hunk 2 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:151]: Thank you!

File: habitat-baselines/README.md (hunk 4 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:212]: Thank you!

File: habitat-baselines/README.md (hunk 5 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:211]: Thank you!

File: habitat-baselines/README.md (hunk 6 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md:222]: Thank you!

File: habitat-baselines/README.md (hunk 7 above)
Comment (jimmytyyang) [reply habitat-baselines/README.md]: Thank you!

**Review by xavierpuigf — APPROVED** [review]
thanks Jimmy! Looks good to me
